Makefile

```diff
@@ -123,14 +141,60 @@ $(OC) oc: $(TOOLS_DIR)
 		$(OC) version | grep -q $${version##v} ;\
 	}
 
+.PHONY: jsonnet
+$(JSONNET) jsonnet: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		export GOBIN=$(TOOLS_DIR) ;\
+		[[ -f $(JSONNET) ]] ||  go install github.com/google/go-jsonnet/cmd/jsonnet@latest ;\
+	}
+
+.PHONY: jsonnetfmt
+$(JSONNETFMT) jsonnetfmt: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		export GOBIN=$(TOOLS_DIR) ;\
+	  [[ -f $(JSONNETFMT) ]] || go install github.com/google/go-jsonnet/cmd/jsonnetfmt@latest ;\
```

Comment:
these one liners don't actually need `@{ ... }` hack and can be simplified as follows
```
target: deps 
  GOBIN=$(TOOLS_DIR) go install .... 
```

`[[ -f ]]` is superfluous since `go install` will take care of that.